.github/workflows/ci_accountability.yml

```
@@ -21,17 +21,29 @@ on:
       - "decidim-participatory_processes/**"
       - "decidim-proposals/**"
 
+# https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#permissions
+permissions:
+  actions: none
+  attestations: none
+  checks: none
+  contents: read
+  deployments: none
+  id-token: none
+  issues: none
+  discussions: none
+  packages: none
+  pages: none
+  pull-requests: read
+  repository-projects: none
+  security-events: none
+  statuses: none
```

```suggestion
permissions:
  contents: read
  pull-requests: read
```

According to the docs:

> If you specify the access for any of these permissions, all of those that are not specified are set to none. 

So, let's remove the ones that we don't need (as having `none` in 20 files is easy to miss a `write` in one of those